.gitignore

```diff
@@ -1,5 +1,5 @@
 node_modules/
-lib/
+typechain/
```

Comment:
I changed it because `lib/` is a reserved folder name in typescript and I thought it was responsible for the issue below (it wasn't but at the time I had already changed it to try so just kept it)